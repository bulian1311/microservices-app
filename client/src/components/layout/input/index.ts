import { Input } from "./input.styles";

export default Input;
