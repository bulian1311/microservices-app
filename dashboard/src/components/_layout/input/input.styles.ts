import styled from "styled-components";

export const Input = styled.input`
  width: 100%;
  background-color: var(--bg-color);
  box-shadow: inset 3px 3px 10px #c6c6c6, inset -3px -3px 10px #ffffff;
  border-radius: 4px;
  padding: 12px 1.05em;
  font-family: inherit;
  font-size: 1.3125rem;
  line-height: 1.19em;
  font-weight: 100;
`;
