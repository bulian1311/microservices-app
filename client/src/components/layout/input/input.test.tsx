import * as React from "react";
import { render, RenderResult } from "@testing-library/react";
import { Input } from "./input.styles";

let documentBody: RenderResult;

beforeEach(() => {
  documentBody = render(<Input />);
});

test("TextArea matces snapshot.", () => {
  const { baseElement } = documentBody;
  expect(baseElement).toMatchSnapshot();
});
