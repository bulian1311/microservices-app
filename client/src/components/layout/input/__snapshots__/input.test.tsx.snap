// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea matces snapshot. 1`] = `
<body>
  <div>
    <input
      class="sc-bdfBwQ dnyjqM"
    />
  </div>
</body>
`;
